app/gui2/src/components/GraphNode.vue

```diff
@@ -4,7 +4,7 @@ import { Rect } from '@/stores/rect'
 import { usePointer, useResizeObserver } from '@/util/events'
 import { computed, onUpdated, reactive, ref, watch, watchEffect } from 'vue'
 import NodeSpan from './NodeSpan.vue'
-import type { ContentRange, ExprId } from '../../shared/yjs-model'
+import type { ContentRange, ExprId } from 'shared/yjs-model'
```

Comment:
in one of the PRs i used `@shared` instead. not sure how conventional it is, and it's not like `@shared/package-name-here` isn't a valid identifier for a npm package. it *may* potentially make it more obvious that it might be a local package?

Reply:
It is a project-local path. It was already working in vue files that way. I have only used the relative paths previously because vite had an issue finding those for some reason when declared inside ts files. So I have added the "alias" to make it understand that. `@` is a convenient alias, because we are using it everywhere. I don't think we need one for `shared`.

Reply:
ah... TIL